### PR TITLE
#15561: supported copy button in query editor

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes [#15561](https://github.com/open-metadata/OpenMetadata/issues/15561)

- supported `copy button in query editor` and also direct copy from the editor
- also supported copy from the editor for `storedProcedure` and `dashboardDataModel` in the `sql and code tab`.

image
image

https://github.com/open-metadata/OpenMetadata/assets/66266464/e04bffaa-e0ba-4a42-954b-3150039a797c

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@Ashish8689, I noticed that you removed this prop from several places. Could you please let me know the reason behind it?

```js
options={{
            readOnly: false,
          }}
```

**Comment by Ashish8689** [comment]
yes, so we have this props already in the defaultOptions  for SchemaEditor. so don't need to pass it!
image

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya ,  so we have this props already in the defaultOptions  for SchemaEditor. so don't need to pass it!
> 
> image
> 
> 

Got it, thanks.

File: openmetadata-ui/src/main/resources/ui/src/pages/AddQueryPage/AddQueryPage.component.tsx [hunk 1]
```diff
@@ -227,9 +227,7 @@ const AddQueryPage = () => {
                   <SchemaEditor
                     className="custom-query-editor query-editor-h-200 custom-code-mirror-theme"
                     mode={{ name: CSMode.SQL }}
-                    options={{
-                      readOnly: false,
-                    }}
+                    showCopyButton={false}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/AddQueryPage/AddQueryPage.component.tsx:230]: @Ashish8689 , I think we can have default option as a false in the editor, so we only have to pass this prop if we want to make it true, wdyt?

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/AddQueryPage/AddQueryPage.component.tsx:230]: so, the idea was to have the copy options as always where ever we are using editor and only some places we don't want that copy button. That the reason we have set by default as true and based on your requirement u can set it to false.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**